backend/src/main/java/de/neuefische/backend/service/UtilService.java

```diff
@@ -1,25 +1,62 @@
 package de.neuefische.backend.service;
 
+import de.neuefische.backend.dto.AppointmentDto;
 import de.neuefische.backend.dto.DoctorDto;
+import de.neuefische.backend.model.Appointment;
 import de.neuefische.backend.model.Doctor;
 import org.springframework.stereotype.Service;
 
+import java.util.List;
+
 @Service
 public class UtilService {
```

Comment:
I think a better name for this class would be DoctorMapper  it doesn't do anything but map doctors.
all the methods could be static, so you wouldn't have to instantiate the class, but could rather import the methods straight away